ops/cmd/start.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"strings"
+	"cybele/ops/connect"
+	"cybele/ops/utils"
+	"fmt"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(startCmd)
+}
+
+// listCmd lists the currrent queue of torrents added
+var startCmd = &cobra.Command{
+	Use:   "start",
+	Short: "List items in current queue of torrents to download",
+	Run:   RunStartCmd,
+	Args:  cobra.RangeArgs(1, 20),
```

Comment:
Since the args ranges between 1-20, the StartCmd should have some awareness for multipe args. Right now it only parses 1 args (i.e one torrent file)

Reply:
For now I have kept it like this because 1, I didn't know how to keep it like accepting strings of any length, and 2, since probably we'll use the auto-complete if we could find the same here, I decided to temporarily put it like this.

Reply:
Got it. Merged